src/pybind/mgr/cephadm/ceph_volume.py

```diff
@@ -0,0 +1,427 @@
+from cephadm.serve import CephadmServe
+from typing import List, TYPE_CHECKING, Any, Dict, Set, Tuple
+if TYPE_CHECKING:
+    from cephadm import CephadmOrchestrator
+
+
+class CephVolume:
+    def __init__(self, mgr: "CephadmOrchestrator", _inheritance: bool = False) -> None:
+        self.mgr: "CephadmOrchestrator" = mgr
+        if not _inheritance:
```

Comment:
I'm not a huge fan of this concept. I get that it's so that this class can be created in certain spots without a circular dependency, but imo we should just have an instance of this class as an attribute of the `CephadmOrchestrator` class and just always invoke it through that. There's a lot of precedence for that concept in cephadm i.e. most classes in inventory.py